### PR TITLE
IGAPP-958: Add more ukrainian translations

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: translations/store-translations/integreat.json [hunk 1]
```diff
@@ -95,13 +95,16 @@
...
-      "defaultReleaseNote": "Wir haben die App weiter für Sie verbessert und einige Fehler entfernt. Helfen Sie mit Integreat zu verbessern, indem Sie Feedback zur App und den lokalen Informationen geben"
+      "defaultReleaseNote": "Wir haben die App weiter für Sie verbessert und einige Fehler entfernt. Helfen Sie mit Integreat zu verbessern, indem Sie Feedback zur App und den lokalen Informationen geben."
```
Comment (SteffiStuffel) [inline translations/store-translations/integreat.json:107]: Have you deleted this dot on purpose?

File: translations/store-translations/integreat.json [hunk 2]
```diff
@@ -177,6 +180,11 @@
...
+      "defaultReleaseNote": "Ми додали новинки та виправили деякі помилки. Допоможіть нам поліпшити Інтегрейт, залишивши свій відгук у додатку"
```
Comment (SteffiStuffel) [inline translations/store-translations/integreat.json:187]: Also dot missing in the end if not intended.

**Review by SteffiStuffel — APPROVED** [review]

File: translations/store-translations/integreat.json (hunk 1 above)
Comment (klinzo) [reply translations/store-translations/integreat.json:107]: I think I added the dot, didn't I?

File: translations/store-translations/integreat.json (hunk 2 above)
Comment (klinzo) [reply translations/store-translations/integreat.json:187]: Good catch :+1:

File: translations/store-translations/integreat.json (hunk 1 above)
Comment (SteffiStuffel) [reply translations/store-translations/integreat.json:107]: Yes you did, sometimes my brain doesn't think green and red are intuitive for adding and removing :D

**Review by f1sh1918 — APPROVED** [review]

**Pull request merged**